src/gui/rss/automatedrssdownloader.cpp

```diff
@@ -423,15 +425,40 @@ void AutomatedRssDownloader::on_exportBtn_clicked()
 
 void AutomatedRssDownloader::on_importBtn_clicked()
 {
+    const QString ALL_SUPPORTED_RULES = "All supported formats (*.rssrules *.filters *.dat)";
+    const QString QBITTORRENT_RULES = "qBittorent (*.rssrules *.filters)";
+    const QString UTORRENT_RULES = "uTorrent (*.dat)";
+    const QString DIALOG_FILTER = ALL_SUPPORTED_RULES % ";;" % QBITTORRENT_RULES % ";;" % UTORRENT_RULES;
+
     // Ask for filter path
-    QString load_path = QFileDialog::getOpenFileName(this, tr("Please point to the RSS download rules file"), QDir::homePath(), tr("Rules list") + QString(" (*.rssrules *.filters)"));
+    QString selectedFilter = ALL_SUPPORTED_RULES;
+    QString load_path = QFileDialog::getOpenFileName(this, tr("Select RSS rules file"), QDir::homePath(), DIALOG_FILTER, &selectedFilter);
```

Comment:
I've read the style guide, I was just trying to minimize my changes to existing code except where absolutely necessary. If the your policy is to fix the style of anything you have to touch, I'll make another pass over my changes and cleanup any pre-existing style issues as well.

Before I do another push though, does anybody have additional non-style related concerns? I'd like to address all remaining issues with my next push if possible.